src/components/StartMenu/StartMenu.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Button, Dropdown, Icon } from 'semantic-ui-react';
+
+const startMenu = () => (
```

Comment:
All component names need to be capitalized and camel-case. So it should be named `StartMenu`.